devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -187,6 +187,10 @@ def getYaleDefaults() {
 	 4: false]
 }
 
+def getEverspringDefaults() {
+	[1: 180]
+}
```

Comment:
I'd just make this a `getEverspringDefaultAlarmLength()` since it's the only one